airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -93,36 +166,86 @@ def poll_job_completion(self):
             if failed_job_count != 0:
                 raise Exception('%s has failed pods, failing task.' % self.job_name)
 
+    def create_job_yaml(self, context):
+        """
+        create job_yaml_string from the operator's parameters
+
+        :param context: Anything that implements xcom_pull
+        :return: A tuple of the job's unique name and a string of YAML for the job
+        """
+        #
+        unique_job_name = "-".join([
```

Comment:
if this was pulled into a separate function, we could test it more explicitly

Reply:
done